class1/README.md

```diff
@@ -1 +1,131 @@
+- design/scalability
+- AWS/GC/Heroku deployment
+- Unit testing
+- Security
+- Data Streams/Sockets
+- APIs
+- Architecture
+- Diagramming (UML, flow, etc.)
+- Dependency Management
+- Code Structure
+
+##Introduction to the Internet:
+![Series of tubes](http://crackerdiet.net/pub/Tedstevenstubesomgwhat.jpg)
+
+This is obviously a comic misconception about the true nature of the internet, so lets dive deeper and have a look at the true nature of our old friend. 
+
+###Browser
+Browsers are a form of connection to the internet that we are all very familiar with. They serve as a very user friendly way to make a **request** to various servers around the world and then display the **response** to those requests. The browser is one example of a **client**, which is something that sends requests to a **server**. HTTP is simply a mutually agreed upon protocol that clients and servers can use to communicate effectively between one another. 
+
+![Request response diagram](http://docstore.mik.ua/orelly/weblinux2/modperl/figs/pmp_0101.gif)
+##Server Structure
+The server's job is to be constantly listening for incomming connections, accept those connections, do some processing, and send back a response. The task of elegantly accepting connections, processing them, and sending a response can be a large and complex. Luckily, there are many excellent existing and open-source frameworks for us to choose from to do much of the work for us. There are also additional integration challenges, such as connecting to databases, that we can make significantly easier by using other frameworks. Throughout the last few decades, several different combinations of these frameworks have been popular and whichever combination you choose to use is called your **Server Stack**. Some such popular server stacks get cute names such as LAMP (Linux, Apache, MySQL, and PHP) and MEAN (MongoDB, Express, Angular, and Node). Throughout this course we will, as the name suggests, be focusing on implementing server architectures that utilize Node.js, which forms the backbone of the server. 
+
+##Javascript
+Many of you are likely familiar with Javascript, but don't worry if you aren't. Javascript, despite what the name suggests, has almost no connection to Java. It is the only actual programming language that runs within your browser itself. syntactically, it is probably one of the more *loose* programming languages and looks similar to python. It was originally designed for being used for quick, small, lightweight tasks within browsers and has since grown into its own language. Only in relatively recent years have personal computers and browsers gotten good enough to safely be able to run more serious Javascript code. It is a scripted language (meaning it executes line by line rather than compiling the whole program first) and has a relatively unique object-oriented implementation. You will get to hear A LOT more about Javascript in the coming weeks, so we will leave it here for now. 
```

Comment:
javascript doesn't have anything at all to do with java (beyond the name).
Might want to mention that is it the relative slowness of javascript that (while improving quickly) was the limiting factor for many years.
Maybe mention ECMAscript? or at least that we will teach ES5, a modern release of javascript with some useful language additions.